iOS/dough-bros-ios/View/Add Group/AddGroupViewController.swift

```diff
@@ -75,18 +75,19 @@ class AddGroupViewController: UIViewController, UITextFieldDelegate {
         dismiss(animated: true, completion: nil)
         print("next tapped")
         
+        // group_id is not actually used by createGroup, so dummy value of 0 used here
         let groupObj = GroupObj(group_id: 0, group_name: "", creator_id: (Auth.auth().currentUser?.uid)!, image_uri: "", amount: 0.0)
         let newGroupID = GroupEndpoints.createGroup(group: groupObj)
         
         for user in group {
             GroupEndpoints.addUserToGroup(user: user, groupID: newGroupID, addedBy: (Auth.auth().currentUser?.uid)!)
         }
         
-        let detailedGroupsVC = GroupDetailsViewController()
-        /// TODO: fetch GroupObj by newGroupID, and set detailedGroupsVC.groupObc
+//        let detailedGroupsVC = GroupDetailsViewController()
+//        /// TODO: fetch GroupObj by newGroupID, and set detailedGroupsVC.groupObc
         
         navigationController?.popViewController(animated: true)
-        navigationController?.pushViewController(detailedGroupsVC, animated: true)
+//        navigationController?.pushViewController(detailedGroupsVC, animated: true)
```

Comment:
temporary fix to force unwrap of groupObj inside detailedGroupsVC causing app to crash. will need to update with return value of createGroup API call